extra-cpp-bindings/src/lib.rs

```diff
@@ -47,6 +48,12 @@ mod ffi {
         fn set_handshaketopic(self: Pin<&mut WalletConnectSessionInfo>, handshaketopic: String);
     }
 
+    pub struct WalletQrcode {
+        pub qrcode: String,
+        pub image: Vec<u8>, /* size* size*/
+        pub size: u32,
```

Comment:
Is the size needed? One will be able to get the length from the C++ vector method?

Reply:
size is necessary to iterate x,y in the image

Reply:
So it's the dimension, not the length